src/screens/Lessons/screens/Lesson/components/LessonData/index.js

```diff
@@ -56,25 +60,49 @@ export default ({lesson}) => {
       title: videoTitleText,
       children: lesson.wistia_id
         ? <WistiaVideo wistiaId={lesson.wistia_id} />
-        : null
+        : isInstructorsOwnLesson 
+          ? <Anchor url={lesson.lesson_http_url}>
+              <Button type='primary'>
+                {uploadVideoTitleText}
+              </Button>
+            </Anchor>
+          : noVideoDescriptionText
```

Comment:
Adds a temporary "Upload" button that links to the lesson edit page on egghead.io, until we build a queue in the Instructor Center directly